slack/web/base_client.py

```diff
@@ -35,7 +35,7 @@ def __init__(
         session=None,
         headers: Optional[dict] = None,
     ):
-        self.token = token.strip()
+        self.token = None if token is None else token.strip()
```

Comment:
Newly added unit tests detected this `strip()` method call (added by #599) doesn't work for the case where `token` is `None`.